queue_services/entity-emailer/src/entity_emailer/email_processors/__init__.py

```diff
@@ -22,17 +22,18 @@
 from typing import Tuple
 
 import requests
-from entity_queue_common.service_utils import logger
 from flask import current_app
-from legal_api.models import Business, Filing
+from flask import request
+from legal_api.models import LegalEntity, Filing
 from legal_api.utils.legislation_datetime import LegislationDatetime
+from entity_emailer.services.logging import structured_log
 
 
 def get_filing_info(filing_id: str) -> Tuple[Filing, dict, dict, str, str]:
     """Get filing info for the email."""
     filing = Filing.find_by_id(filing_id)
     if filing.business_id:
-        business = Business.find_by_internal_id(filing.business_id)
+        business = LegalEntity.find_by_internal_id(filing.business_id)
```

Comment:
Should the variable be renamed to "entity" (or something similar)? Like: `entity = ` instead of `business = `

Reply:
i think it's fine for now to leave it as `business`